cryptotik/hitbtc.py

```diff
@@ -30,44 +61,161 @@ def api(cls, url):
     def get_market_ticker(cls, pair):
         '''returns simple current market status report'''
 
-        return cls.api(cls.url + cls.format_pair(pair) + "/ticker")
+        return cls.api(cls.url + "public/"+ "ticker/" + cls.format_pair(pair))
 
     @classmethod
-    def get_market_trade_history(cls, pair):
+    def get_market_trade_history(cls, pair, limit=1000):
         '''get market trade history'''
 
-        return cls.api(cls.url + cls.format_pair(pair) \
-        + "/trades/recent?max_results=1000&format_item=object")
+        return cls.api(cls.url + "public/trades/" + cls.format_pair(pair), 
+                        params={'limit': limit})
 
     @classmethod
-    def get_market_order_book(cls, pair):
+    def get_market_orders(cls, pair):
         '''return order book for the market'''
 
-        return cls.api(cls.url + cls.format_pair(pair) \
-        + "/orderbook?format_price=number&format_amount=number")
+        return cls.api(cls.url + "public/orderbook/" + cls.format_pair(pair))
 
     @classmethod
     def get_market_spread(cls, pair):
         '''return first buy order and first sell order'''
 
+<<<<<<< HEAD
+=======
         from decimal import Decimal
 
-        order_book = cls.get_market_order_book(pair)
+>>>>>>> ae66876... Hitbtc: updated to new API, added private methods and test
+        order_book = cls.get_market_orders(cls.format_pair(pair))
 
-        ask = order_book["asks"][0][0]
-        bid = order_book["bids"][0][0]
+        ask = order_book['ask'][0]['price']
+        bid = order_book['bid'][0]['price']
 
         return Decimal(ask) - Decimal(bid)
 
+
+    @classmethod
+    def get_markets(cls):
+        '''Find supported markets on this exchange'''
+
+        r = cls.api(cls.url + "public/" + "symbol")
+
+        pairs = [i["id"].lower() for i in r]
+        return pairs
+
+    @classmethod
+    def get_market_volume(cls, pair):
+        ''' return volume of last 24h'''
+
+        return cls.get_market_ticker(cls.format_pair(pair))["volume"]
+
     @classmethod
     def get_market_depth(cls, pair):
         '''return sum of all bids and asks'''
 
+<<<<<<< HEAD
```

Comment:
You should not be having this <<<<HEAD artifacts from git, you did something wrong with your merge.